src/cordon/cli.py

```diff
@@ -55,6 +55,18 @@ def parse_args() -> argparse.Namespace:
         default=2048,
         help="Context size for llama.cpp (default: 2048)",
     )
+    backend_group.add_argument(
+        "--api-key",
+        type=str,
+        default=None,
+        help="API key for remote embeddings (remote backend only, falls back to env vars)",
+    )
+    backend_group.add_argument(
+        "--endpoint",
+        type=str,
```

Comment:
Minor: For consistency with other remote backend options (`--api-key`, `--endpoint`), consider adding a `--request-timeout` CLI flag. Users may need to increase the timeout for slow connections or large batches.

Currently the README shows this as `N/A` in the CLI column.